src/itensor.jl

```diff
@@ -319,7 +311,7 @@ emptyITensor() = emptyITensor(Float64)
 Construct an ITensor with empty storage and `Any` number of indices.
 """
 function emptyITensor(::Type{ElT}, ::Type{Any}) where {ElT <: Number}
```

Comment:
We'll probably remove this constructor (see https://github.com/ITensor/ITensors.jl/pull/591/files#r594604222).